userspace/libsinsp/include/sinsp_external_processor.h

```diff
@@ -1,4 +1,6 @@
 #pragma once
+#include "sinsp.h"
```

Comment:
Instead of having a header include a header here, can we just forward declare `sinsp`?

```c++
class sinsp;
```

Reply:
yes that's a good idea since then if you don't override that method you don't need to pull in the whole thing.

In reality most users will pull it in anyway, but I agree. will do.